XenModel/FriendlyNames.resx

```diff
@@ -1076,6 +1076,15 @@
   <data name="Message.body-vcpu_qos_failed" xml:space="preserve">
     <value>Quality of Service settings for VCPU '{0}' on virtual machine '{1}' could not be obeyed.</value>
   </data>
+  <data name="Message.body-vdi_cbt_metadata_inconsistent" xml:space="preserve">
+    <value>Changed block tracking is disabled for virtual disk {0} due to inconsistent metadata. To continue using changed block tracking with the virtual disk, you will need to do the backup setup again, including re-enabling changed block tracking, taking a snapshot and backing it up.</value>

```

Comment:
... **has been** disabled...
... with **this** vistual disk .....

... due to **a** resizing failure ...
... due to **a** snapshot failure ...